<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideShowDlg</class>
 <widget class="QDialog" name="SlideShowDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="7,1">
   <item>
    <widget class="QWidget" name="slideShow" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="PicAnimationWidget" name="picAnimation" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="slideNextWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="picButton" name="slideNextBtn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="slidePreWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="picButton" name="slidePreBtn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="PicStateButton" name="playBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="picButton" name="pauseBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="preShow" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155161</width>
          <height>120</height>
         </size>
        </property>
        <widget class="PriListWid" name="preListWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1677215</width>
           <height>110</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>picButton</class>
   <extends>QPushButton</extends>
   <header>picbutton.h</header>
  </customwidget>
  <customwidget>
   <class>PicStateButton</class>
   <extends>QPushButton</extends>
   <header>picstatebutton.h</header>
  </customwidget>
  <customwidget>
   <class>PicAnimationWidget</class>
   <extends>QWidget</extends>
   <header>picanimationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PriListWid</class>
   <extends>QListWidget</extends>
   <header>prilistwid.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
